pkgs/tools/security/sbomnix/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, coreutils
+, curl
+, gnugrep
+, gnused
+, gzip
+, nix
+, python3Packages
+  # python libs
+, colorlog
+, graphviz
+, numpy
+, packageurl-python
+, pandas
+, requests
+, reuse
+, tabulate
+}:
+
+python3Packages.buildPythonApplication rec {
```

Comment:
```suggestion
python.pkgs.buildPythonApplication rec {
```

Reply:
Fixed in 8fae5ee3213024a3f498ebccbc1e94ac4a77ef8e